pkg/controller/servicebindingrequest/secret_test.go

```diff
@@ -51,4 +50,10 @@ func TestSecretNew(t *testing.T) {
 		assert.True(t, found)
 		assertSecretNamespacedName(t, u, ns, name)
 	})
+
+	t.Run("Delete", func(t *testing.T) {
+		err := s.delete()
+		assert.NoError(t, err)
+	})
+
```

Comment:
see above comment. We do not use `delete()` anywhere in production code, so let's remove it along with the test.